src/index.pug

```diff
@@ -226,28 +226,28 @@ html
 
     footer
       .container
-        ul.footer-links 
-          li 
+        ul.list-unstyled.footer-links.row.justify-content-center
+          li.col-auto
```

Comment:
Do we need to assign `<li>` `.col-auto` class here? Layout seems fine without those extra rules in `col-auto`. Or I guess it's a good practice to assign immediate children of a `.row` element with `col-*` class?